kinto_attachment/tests/config/seafile.ini

```diff
@@ -0,0 +1,14 @@
+[app:main]
+use = egg:kinto
+kinto.userid_hmac_secret = some-secret-string
+
+kinto.includes = kinto.plugins.default_bucket kinto_attachment
+
+kinto.attachment.base_url = https://seafile.net
+
+kinto.attachment.folder = {bucket_id}/{collection_id}
+
+kinto.attachment.seafile.servname = https://seafile.net
+kinto.attachment.seafile.username = seafile
```

Comment:
I did it that way because of the seafile python api which works with login/password to open a connection and manage the other parts of the api.